src/main/java/com/projecturanus/betterp2p/client/gui/GuiAdvancedMemoryCard.kt

```diff
@@ -86,7 +86,7 @@ class GuiAdvancedMemoryCard(msg: S2CListP2P) : GuiScreen(), TextureBound {
         )
     }
 
-    val BACKGROUND: ResourceLocation = ResourceLocation(MODID, "textures/gui/advanced_memory_card.png")
+    val background: ResourceLocation = ResourceLocation(MODID, "textures/gui/advanced_memory_card.png")
```

Comment:
This should be pulled out of the class as it really should be a singleton. thanks for the poke